kedro-datasets/RELEASE.md

```diff
@@ -1,12 +1,21 @@
 # Upcoming Release:
 
-## Major features and improvements:
+## Major features and improvements
+
+## Bug fixes and other changes
+
+## Community contributions
+
+# Release 1.3.0:
```

Comment:
```suggestion
# Release 1.3.0
```

?

Reply:
To be consistent with the rest of the file, it looks like `Release x.x.x` should have a colon, no other titles / subtitles should. Now that you mention it, `Release 1.2.0` is missing a colon too :).